implementations.md

```diff
@@ -0,0 +1,11 @@
+# Implementations of Container Specification
+
+## Runtime
+
+* [opencontainers/runc](https://github.com/opencontainers/runc) - reference implementation of OCI runtime
+* [hyperhq/runv](https://github.com/hyperhq/runv) - Hypervisor-based runtime for OCI
+
+## Testing
+
+* [huawei-openlab/ocp-testing](https://github.com/huawei-openlab/ocp-testing) - Testing framework for OCI configuration and runtime
```

Comment:
We are working on  “is this runtime compatible with opencontainers/specs”.

The 'bundle-specification compatibility' is not planed yet.